File: EjerciciosUDEMY/contarcaracter.js
// 2. cuantas veces se repite un caracter

let text = "sadsadsafsddcdvxcgdfsd"
let caracter = "a"

let i = 0;

for (let letra of text)
{
    if (letra == caracter)
    {
        i++;
    }
}

console.log(i)

File: EjerciciosUDEMY/contarcaracter.py
# 2. cuantas veces se repite un caracter

TEXT = "sadsadsafsddcdvxcgdfsd"
caracter = input("ingrese el caracter a buscar: ")

i = 0

for letra in TEXT:
    if letra == caracter:
        i += 1

print(i)